src/lib.rs

```diff
@@ -48,3 +49,40 @@ pub fn calculate_term_width() -> u16 {
         }
     }
 }
+
+#[derive(Default, Debug, Clone, PartialEq, serde_derive::Serialize, serde_derive::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct FilterList {
+    pub ignore: Vec<Ignore>,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, serde_derive::Serialize, serde_derive::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Ignore {
+    pub id: String,
+    pub reason: Option<String>,
+}
+
+pub fn filter_vulnerabilities(packages: &mut Vec<Coordinate>, exclude_vuln_file_path: String) {
+    let filter_list_str = fs::read_to_string(exclude_vuln_file_path).expect("Unable to read file");
+    let filter_list_json: FilterList = serde_json::from_str(&filter_list_str).expect("JSON was not well formatted");
```

Comment:
It would probably be a good idea to do something like [this serde example](https://docs.serde.rs/serde_json/fn.from_reader.html#example) which uses `from_reader` with a `BufReader` to stream the data from disk directly to the parser. It will be _slightly_ more performant

Reply:
In: 546bb80